consensus/polybft/signer/signature_test.go

```diff
@@ -0,0 +1,108 @@
+package bls
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	MessageSize        = 5000
+	ParticipantsNumber = 64
+)
+
+var (
+	validTestMsg   = genRandomBytes(MessageSize)
+	invalidTestMsg = genRandomBytes(MessageSize)
+)
+
+func Test_VerifySignature(t *testing.T) {
+	blsKey, _ := GenerateBlsKey()
+	signature, err := blsKey.Sign(validTestMsg)
+	require.NoError(t, err)
+
+	assert.True(t, signature.Verify(blsKey.PublicKey(), validTestMsg))
+	assert.False(t, signature.Verify(blsKey.PublicKey(), invalidTestMsg))
+}
+
+func Test_AggregatedSignatureSimple(t *testing.T) {
+	bls1, _ := GenerateBlsKey()
+	bls2, _ := GenerateBlsKey()
+	bls3, _ := GenerateBlsKey()
```

Comment:
You can extract a helper method for generating multiple BLS keys

Reply:
In fact we have introduced useful (testValidators) abstraction which can be used here as well:
https://github.com/0xPolygon/polygon-edge/pull/774/files/654491a4c9c4b03dfeae1bf72c66838984610796#diff-40478145f100201698abd4dab75eb2abfaf678152edb38e0e88747348f152f0fR18

Reply:
Fixed through #825